### PR TITLE
S/390 basic documentation

### PR DESCRIPTION
**Pull request by wxwisiasdf**
when the mainframe is sus

Signed-off-by: superleaf1995 <jesusantonio30122016@gmail.com>

### CODE REVIEW

File: modules/ROOT/pages/s390.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
+= S/390
+:description: Description about the 390 architecture
+:keywords: s390, zarch, z11, s360, s370
+:page-category: Hardware
+:source-language: c
+
+== History
+The S/390 is part of the S/3X0 family, which includes the S/360, S/370 and S/390 - all of these were released around in 1960, 1970 and 1990 respectively.
```
Comment (lukflug) [inline modules/ROOT/pages/s390.adoc:8]: Add a lead section, or turn this to the lead section.

File: modules/ROOT/pages/s390.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Allows to unpack a HFP.
+
+=== Vector Instructions (z/Arch only)
+These are not an extension per-se since they are virtually in every z/Arch machine - however they offer vectorization opportunities for compilers and allows for AVX-like operation.
+
+== See also
+
+* https://www.ibm.com/docs/en/SSQ2R2_15.0.0/com.ibm.tpf.toolkit.hlasm.doc/dz9zr006.pdfe[z/Arch Principles of Operation Manual]
+* http://bitsavers.org/pdf/ibm/370/princOps/SA22-7085-1_370-XA_Principles_of_Operation_Jan87.pdf[S/370 Extended Principles Of Operation Manual]
+* https://www.kernel.org/doc/html/v5.3/s390/debugging390.html[Debugging z/Linux on the S/390]
```
Comment (lukflug) [inline modules/ROOT/pages/s390.adoc:70]: Change "see also" to "external links".

File: modules/ROOT/pages/s390.adoc [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+> Word, 32-bits
+> Double word, 64-bits
+> Define Storage (DS on HLASM), Uninitialized data
+> Define Character (DC on HLASM), Initialized data
+> Procedure, Function
+> Dataset, File
+> Processing Unit, CPU
+> Supervisor Call, SYSCALL
+> Principles Of Operation is the main manual and "architecture-reference" document, often abbreviated as POP
+> The word "Recognized" is related to an interrupt being fired up as in: "A MC interrupt is recognized when a device fails"
```
Comment (lukflug) [inline modules/ROOT/pages/s390.adoc:29]: Format this using a [description list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/)

**Comment by wxwisiasdf** [comment]
original fork deleted,

**Pull request closed without merging**